src/dispatching/updater.rs

```diff
@@ -103,40 +102,35 @@ pub trait Updater:
     type Error;
 }
 
-#[pin_project]
-pub struct StreamUpdater<S> {
-    #[pin]
-    stream: S,
+type StreamItem = Result<Update, RequestError>;
+
+struct InnerUpdater<'a> {
+    stream: Box<dyn Stream<Item = StreamItem> + 'a>,
```

Comment:
In the previous implementations, there were no boxes... I think boxing here is useless.

Reply:
There is also a compilation error related to Pin.

Reply:
You didn't compile code before committing?

Reply:
I've compiled, but I don't know how to solve the problem with Pin.

Reply:
So you committed code that doesn't compile or what?

Reply:
Yes, I thought that someone will help.